terraform/aws/analytical-platform-data-engineering-production/ppud-dev/main.tf

```
@@ -0,0 +1,58 @@
+# checkov:skip=CKV2_AWS_62: Skipping because event notifications are not applicable for this bucket.
+# checkov:skip=CKV_AWS_18: Skipping because access logging is managed externally.
+# checkov:skip=CKV_AWS_145: Skipping because KMS encryption is handled at the account level.
+# checkov:skip=CKV2_AWS_61: Skipping because a lifecycle configuration is not required.
+# checkov:skip=CKV_AWS_144: Skipping because cross-region replication is not needed.
+
+
+resource "aws_s3_bucket" "ppud_landing" {
+  bucket              = "ppud-dev"
```

Bucket names are globally unique, can you use something like
```suggestion
  bucket              = "mojap-data-engineering-production-ppud-dev"
```